wiki/Tournaments/Official_support/en.md

```diff
@@ -26,7 +26,7 @@ Community-run tournaments which abide by the following hard criteria are eligibl
 - The tournament series must run no more than four times per year.
 - The tournament must operate on a Round of 16 double-elimination format at a minimum, **OR**, have at least 64 players (or 16 teams) participating. Larger scales and formats are permitted.
 - The tournament is organised and run by an *experienced team of accomplished and reputable staff*,[^experience] **OR**, the tournament *has been run at least once before without receiving rewards*.
-- Staff members must not participate in the tournament themselves at any point, and players are unable to later assist as staff either. For our definition on "staff", please refer to the [FAQ](#what-is-considered-staff).
+- Staff members must not participate in the tournament themselves, though they may join after they have been eliminated. For our definition on "staff", please refer to the [FAQ](#what-is-considered-staff).
```

Comment:
Does this mean that a "player may join as "staff""? If so that needs to be said. Right now it reads as though a staff can join as a player, which I do not believe is the intention.

Reply:
it's even worse since it's not even referring to players. the only thing this sentence is saying is that *staff members may join (the tournament) after being eliminated*, but non-players can't be eliminated so it doesn't make sense.

Reply:
Yeah okay, I think I was way too tired when I approved this suggestion.
Very confusing indeed. 
Fixed that one.

Reply:
I might have been way too tired when approving this one.
Changed wording to make more sense.